test/onnx/test_fx_to_onnx.py

```diff
@@ -77,6 +117,45 @@ def forward(self, x):
         x = torch.arange(1.0, 6.0, requires_grad=True)
         _ = fx_onnx.export(TopKModel(), x, opset_version=self.opset_version)
 
+    def test_unsupported_indices_fake_tensor_generated_with_op_level_debug(self):
+        class EmbedModelWithoutPaddingIdx(torch.nn.Module):
+            def forward(self, input, emb):
+                return torch.nn.functional.embedding(input, emb)
+
+        model = EmbedModelWithoutPaddingIdx()
+        x = torch.randint(4, (4, 3, 2))
+        embedding_matrix = torch.rand(10, 3)
+
+        _ = fx_onnx.export(model, x, embedding_matrix, op_level_debug=True)
+        assert_has_diagnostics(
+            diagnostics.engine,
+            diagnostics.rules.fx_node_to_onnx,
+            diagnostics.levels.WARNING,
+            expected_error_node="aten.embedding.default",
+            expected_error_message="IndexError: index out of range in self",
```

Comment:
@BowenBao A naive assertion from SARIF log. We can have an offline sync if you feel this should be a standalone rule, but if you think it's enough for now, it's fine with me.